lib/spandex_phoenix.ex

```diff
@@ -183,10 +183,7 @@ defmodule SpandexPhoenix do
   def default_metadata(conn) do
     conn = Plug.Conn.fetch_query_params(conn)
 
-    route =
-      conn
-      |> route_name()
-      |> URI.decode_www_form()
```

Comment:
@GregMefford I'm not sure what your test coverage is, but deleting this line didn't break anything? I can bring it back but as `URI.decode()` if you'd like

Reply:
I’ll have to take a look when I get back to a computer. I believe the test coverage has some gaps with respect to how Plug works in test vs. how Cowboy does some of the oddball parsing IRL.